marimo/_plugins/stateless/image.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Marimo. All rights reserved.
+from __future__ import annotations
+
+from typing import Any, Optional
+
+from marimo._output.builder import h
+from marimo._output.hypertext import Html
+from marimo._output.rich_help import mddoc
+from marimo._output.utils import create_style
+
+
+@mddoc
+def image(
+    src: str,
+    alt: Optional[str] = None,
+    width: Optional[int] = None,
+    height: Optional[int] = None,
+    rounded: bool = False,
+    style: Optional[dict[str, Any]] = None,
+) -> Html:
+    """Render an image as HTML.
+
+    **Example.**
+    ```python3
+    mo.image(
+        src="https://marimo.io/logo.png",
+        alt="Marimo logo",
+        width=100,
+        height=100,
+        rounded=True,
+    )
+    ```
+    **Args.**
+
+    - `src`: the URL of the image
```

Comment:
What do you think about letting people use images on their filesystem as well? We could read these as bytes and create a data URL out of them.